.bingo/Variables.mk

```diff
@@ -1,4 +1,4 @@
-# Auto generated binary variables helper managed by https://github.com/bwplotka/bingo v0.8. DO NOT EDIT.
+# Auto generated binary variables helper managed by https://github.com/bwplotka/bingo v0.7. DO NOT EDIT.
```

Comment:
Some changes under the .bingo directory that I don't think are relevant to this pull request.

Reply:
will remove thanks